packages/apps/graph/src/graph/objects/token.ts

```diff
@@ -6,5 +8,21 @@ export default builder.objectType('Token', {
     id: t.exposeID('id'),
     balance: t.exposeInt('balance'),
     chainId: t.exposeInt('chainId'),
+    info: t.field({
+      type: 'TokenInfo',
+      nullable: true,
+      resolve: async (parent) => {
```

Comment:
Could you try catch the whole function with the error normaliser that we also use in the other queries?